aspnetcore/host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps.md

```diff
@@ -0,0 +1,328 @@
+
+#### Test the app
+
+1. Launch the app in Visual Studio. The project includes a Docker file, which means an the arrow next to the run button can be selected to start the app using either a Docker Desktop setup or the standard ASP.NET Core local web server.
+
+Use the search form to browse for GitHub repositories by name.
+
+:::image type="content" source="./media/scaling-app-screenshot.png" alt-text="A screenshot showing the GitHub Explorer app.":::
+
+## Deploy the app to Azure Container Apps
+
+Visual Studio is used to deploy the app to Azure Container Apps. Container apps provide a managed service designed to simplify hosting containerized apps and microservices.
+
+> [!NOTE]
+> Many of the resources created for the app require a location. For this app, location isn't important. A real app should select a location closest to the clients. You may want to select a location near you.
+
+1. In Visual Studio solution explorer, right click on the top level project node and select **Publish**.
+1. In the publishing dialog, select **Azure** as the deployment target, and then select **Next**.
+1. For the specific target, select **Azure Container Apps (Linux)**, and then select **Next**.
+1. Create a new container app to deploy to. Select the green **+** icon to open a new dialog and enter the following values:
+
+    :::image type="content" source="./media/scaling-deploy-visual-studio-small.png" lightbox="./media/scaling-deploy-visual-studio.png" alt-text="A screenshot showing Visual Studio deployment.":::
+
+    * **Container app name**: Leave the default value or enter a name.
+    * **Subscription name**: Select the subscription to deploy to.
+    * **Resource group**: Select **New** and create a new resource group called *msdocs-scalable-razor*.
+    * **Container apps environment**: Select **New** to open the container apps environment dialog and enter the following values:
+        * **Environment name**: Keep the default value.
+        * **Location**: Select a location near you.
+        * **Azure Log Analytics Workspace**: Select **New** to open the log analytics workspace dialog.
+            * **Name**: Leave the default value.
+            * **Location**: Select a location near you and then select **Ok** to close the dialog.
+        * Select **Ok** to close the container apps environment dialog.
+    * Select **Create** to close the original container apps dialog. Visual Studio creates the container app resource in Azure.
+1. Once the resource is created, make sure it's selected in the list of container apps, and then select **Next**.
+1. You'll need to create an Azure Container Registry to store the published image artifact for your app. Select the green **+** icon on the container registry screen. 
+
+    :::image type="content" source="./media/scaling-create-new.png" alt-text="A screenshot showing how to create a new container registry.":::
+
+1. Leave the default values, and then select **Create**.
```

Comment:
<img width="391" alt="image" src="https://user-images.githubusercontent.com/3605364/191112236-c93ef3b4-b8d2-499b-b9d0-a5050714e76a.png">

Reply:
Looks like a bunch of images are incorrect. BTW, we prefer to use absolute links rather than relative. With absolute links, we can copy/paste anywhere in the repo. We can refactor the MD and move them to different directories.

Reply:
Neither of these show as broken for me - could this be a browser issue?

![image](https://user-images.githubusercontent.com/93200798/191113269-54e86dd8-e6e7-40e8-9500-f666913202ae.png)

Reply:
Sometimes images in the preview builds show as broken for me until I refresh a few times.